Sortable.js

```diff
@@ -412,7 +412,12 @@
 				Sortable.active = this;
 
 				// Drag start event
-				_dispatchEvent(this, rootEl, 'start', dragEl, rootEl, oldIndex);
+				if (this.options.forceFallback) {
+					_dispatchEvent(this, rootEl, 'start', ghostEl, dragEl, oldIndex);
```

Comment:
Alas, I can not accept it.
Behavior the library should be the same in both cases.